tools/build/src/commands/docs.ts

```diff
@@ -1,22 +1,25 @@
-import jsdoc2md from 'jsdoc-to-markdown';
-import fs from 'node:fs/promises';
+import fs, { writeFile, mkdir } from 'node:fs/promises';
 import { existsSync } from 'node:fs';
 import path from 'node:path';
-import { writeFile, mkdir } from 'node:fs/promises';
+import jsdoc2md from 'jsdoc-to-markdown';
+import FileSet from 'file-set'
```

Comment:
I chose fileset because it's the same library used by jsdoc2md, so we know the glob will be treated the same way